storage/innobase/que/que0que.cc

```diff
@@ -1233,7 +1233,5 @@ que_eval_sql(
 		mutex_exit(&dict_sys->mutex);
 	}
 
-	ut_a(trx->error_state != 0);
-
```

Comment:
This looks like useless code, trying to catch an uninitialized (by default zero-initialized) `error_state`. This might have been a little more useful before the `trx_pool_t` was introduced. Good riddance.